initializer_inputs/agriculture/agriculture_burner_network_gas_share.ad

```diff
@@ -0,0 +1,4 @@
+- id = 246
```

Comment:
That does not really say anything about the `id` though. But I guess it makes my question irrelevant ;-)